downstream/modules/aap-on-azure/proc-azure-update-route-tables.adoc

```diff
@@ -32,13 +32,13 @@ The name of the {PlatformNameShort} route table uses the following convention:
 ----
 aks-agentpool-<numbers>-routetable
 ----
-. From the route table menu bar, click *Routes* > *Add*.
+. From the route table menu bar, click btn:[Routes] > btn:[Add].
```

Comment:
This is a menu selection and I would rewrite to:

From the route table menu bar, select menu:Routes[Add].